src/w3c/headers.js

```diff
@@ -273,6 +273,9 @@ export function run(conf) {
     }
   }
   conf.title = document.title || "No Title";
+  if (document.title && conf.isPreview && conf.prNumber) {
```

Comment:
I'm unsure about the first check for "document.title"?

Reply:
If I don't check this, and document.title is undefined, I would be setting document.title to be equal to "Preview of PR #XYZ: undefined". But if I leave it undefined when it is undefined, it would be set by  https://github.com/w3c/respec/blob/3c0dda705c7de88f6ec68a913d4b34d8126964d0/src/w3c/templates/headers.js#L27

Reply:
ok, thanks for clarifying. I was unsure.